### PR TITLE
[keys] use prettykeys internally for input bindings and other getkeystroke uses

### PR DESCRIPTION
**Pull request by saulpw**
Replace legacy ^X with Ctrl+X and other curses KEY_ with internal prettykeys.

Does not replace bindings for commands, but everything else should use be using prettykeys now.

### CODE REVIEW

File: visidata/_input.py [hunk 1]
```diff
@@ -227,50 +227,50 @@ def handle_key(self, ch:str, scr) -> bool:
...
+        elif ch == 'Ctrl+G':
             vd.cycleSidebar()
             return False # not considered a first keypress
-        elif ch in ('^H', 'KEY_BACKSPACE', '^?'):  i -= 1; v = delchar(v, i)
-        elif ch == '^I':                           v, i = self.completion(v, i, +1)
-        elif ch == 'KEY_BTAB':                     v, i = self.completion(v, i, -1)
-        elif ch in ['^J', '^M']:                   return True # ENTER to accept value
-        elif ch == '^K':                           v = v[:i]  # ^Kill to end-of-line
-        elif ch == '^N':
+        elif ch in ('Ctrl+H', 'Bksp', 'Ctrl+?'):   i -= 1; v = delchar(v, i)
```
Comment (anjakefala) [inline visidata/_input.py:267]: I got "no command for Ctrl+?" when I typed Backspace

**Comment by anjakefala** [comment]
I also tested `Shift+Z` and `Shift+F` and neither are working! I believe none of the Shifts are working.

Spacebar is also not working.

File: visidata/mainloop.py [hunk 2]
```diff
@@ -212,7 +212,6 @@ def mainloop(vd, scr):
                 vd.warning('duplicate prefix: ' + keystroke)
                 vd.keystrokes = ''
             else:
-                keystroke = vd.prettykeys(keystroke)
```
Comment (anjakefala) [inline visidata/mainloop.py:217]: It's the removal of this line that broke spacebar (`' '`) and the shift keys!
